packages/block/src/block.ts

```diff
@@ -261,24 +262,6 @@ export class Block {
     return stringError ? errors : errors.length === 0
   }
 
-  /**
-   * Performs the following consistency checks on the block:
-   *
-   * - Value checks on the header fields
-   * - Signature and gasLimit validation for included txs
-   * - Validation of the tx trie
-   * - Consistency checks and header validation of included uncles
-   *
-   * Throws if invalid.
-   *
-   * @param blockchain - validate against an @ethereumjs/blockchain
-   * @param onlyHeader - if should only validate the header (skips validating txTrie and unclesHash) (default: false)
-   */
-  async validate(blockchain: Blockchain, onlyHeader: boolean = false): Promise<void> {
-    await this.header.validate(blockchain)
-    await this.validateUncles(blockchain)
-    await this.validateData(onlyHeader)
-  }
```

Comment:
I think I wrote otherwise in the original issue but I guess we really want to keep this combined version (which would also solve a related client-TODO) in a public `Blockchain.validateBlock()` method.

Reply:
The thought being we just want to be able to validate a new block in it's entirety without having to actually put it in the chain?

Reply:
(or even better in the cases where this is possible: both doesn't need to be in the chain, so these would be the cases where a method could even be called in a static context)

Reply:
I've added a public `blockchain.validateBlock` method that wraps up those three validation checks.  I don't think we can make it a static method without breaking it into further pieces since some the checks are heavily dependent on the existence of the chain.